### PR TITLE
Revamp the React apprentice and journeyman badges to reflect experien…

### PR DESCRIPTION
**Pull request by marybeshaw**
…ce the org has gained with our libraries over the past few years.  Go more in depth and provide more online resources.

### CODE REVIEW

**Comment by tylergraf** [comment]
This is looking good!

File: badges-active/react/_mocro_EXPLORER.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+React - EXPLORER [Micro Badge]
```
Comment (spacerockzero) [inline badges-active/react/_mocro_EXPLORER.md:1]: ^^file spelling

**Comment by spacerockzero** [comment]
This is looking really good. You know your stuff, and this should make more like you around here. Great and needed update!

File: badges-active/react/_mocro_EXPLORER.md (hunk 1 above)
Comment (marybeshaw) [reply badges-active/react/_mocro_EXPLORER.md]: Thanks for catching that. I fixed it.

**Review by matthewmatician — APPROVED** [review]

**Review by joeycozza — APPROVED** [review]

**Pull request merged**